src/components/index/DeFiChainEcoSystemSection.tsx

```diff
@@ -49,7 +49,7 @@ export function DeFiChainEcoSystemSection(): JSX.Element {
             </span>
           </span>
         </div>
-        <div className="lg:block lg:h-[639px] lg:place-self-end">
+        <div className="grid grid-flow-row grid-cols-1 gap-6 md:grid-flow-row md:grid-cols-2 lg:block lg:h-[639px] lg:place-self-end">
```

Comment:
check spacing on tablet

Reply:
> Try to refresh when you are at 768px width

Hmm I don't get that, can pull and try again?